kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks-inputs.sh

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# The version of this inputs file. Do not modify.
+export version=create-domain-on-aks-inputs-v1
+
+#
+# Parameters that must be changed from these values!
+#
+
+# Oracle Single Sign-On (SSO) account email, used to pull the WebLogic Server image.
+export dockerEmail=docker-email
+
+# Oracle SSO account password, used to pull the WebLogic Server image.
+export dockerPassword=docker-password
```

Comment:
is this a hard coded password?

Reply:
Even though the file `create-domain-on-aks-inputs.sh` has a `.sh` extension, its main purpose is to serve as a **configuration** file. 
Users can modify certain configuration options within the file, and if they don't modify the parameters, the values within it serve as default values during startup.

Reply:
No, it is not a hard coded password. The user must change it.